.clang-tidy

```diff
@@ -60,6 +60,7 @@ Checks: '*,
          -readability-implicit-bool-conversion,
          -readability-magic-numbers,
          -readability-named-parameter,
+         -readability-convert-member-functions-to-static,
```

Comment:
Not sure if this is the right approach to fix the clang-tidy error? When I run
```
clang-tidy -p build src/storage/v2/replication/replication_server.cpp -- -I./src -I/opt/toolchain-v4/include -I./libs/spdlog/include -std=c++20
```
There are a bunch of warnings but no errors...

What's the best approach here?

Reply:
So first of all, you don't have to fix all the warnings. If you check [our CI](https://github.com/memgraph/memgraph/blob/b782271be87a04173ebe7c724358ba107b15dd9a/.github/workflows/diff.yaml#L114-L115), then you can see only the modified lines are checked in the codebase. So if you don't modify a specific function/line, you don't have fix that. However if you touch it, then the result should be clang-tidy warning free. I think you can use the same command that we are using in our CI to check the warnings locally.

After pointing this out, I think it understandable why I am against of disabling this check. This check is quite reliable and useful. Why not to make such functions static member functions?